test/format/config.c

```diff
@@ -1368,9 +1374,11 @@ config_map_checksum(const char *s, u_int *vp)
     if (strcmp(s, "on") == 0)
         *vp = CHECKSUM_ON;
     else if (strcmp(s, "off") == 0)
-        *vp = CHECKSUM_ON;
+        *vp = CHECKSUM_OFF;
```

Comment:
Yikes! I guess we weren't testing what we thought we were testing.

Reply:
Yeah, that was a shocker for me, too.